libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeViewRenderer/TreeViewRenderer.tsx

```diff
@@ -162,49 +147,37 @@ export class TreeViewRenderer extends React.PureComponent<
       });
 
       svg.style("pointer-events", "all").call(zoom as any);
-
-      //   if (this.state.transform) {
-      //     var x = this.state.transform.x;
-      //     var y = this.state.transform.y;
-      //     var k = this.state.transform.k;
-      //     select(svgOuterFrame.current).call(
-      //       zoom.transform as any,
-      //       d3zoomIdentity.translate(x, y).scale(k)
-      //     );
-      //   }
     }
 
-    const minColor = ColorPalette.MinColor;
-    const maxColor = ColorPalette.MaxColor;
-
-    const colorgrad = d3scaleLinear<string>()
-      .domain([min, max])
-      .interpolate(d3interpolateHcl)
-      .range([minColor, maxColor]);
-
-    const linkVertical = d3linkVertical<any, HierarchyPointNode<any>>()
-      .x((d: any) => d!.x!)
-      .y((d: any) => d!.y!);
+    const linkVertical = d3linkVertical<any, HierarchyPointNode<ITreeNode>>()
+      .x((d: HierarchyPointNode<ITreeNode>) => d!.x!)
+      .y((d: HierarchyPointNode<ITreeNode>) => d!.y!);
     // GENERATES LINK DATA BETWEEN NODES
     // -------------------------------------------------------------------
-    const links = rootDescendents.slice(1).map((d) => {
-      const thick = 1 + Math.floor(30 * (d.data.size / this.state.rootSize));
-      const lineColor = d.data.onSelectedPath ? "#089acc" : "#e8eaed";
-      const id: string = d.id!;
-      const linkVerticalD = linkVertical({ source: d.parent, target: d });
-      return {
-        d: linkVerticalD!,
-        id: id + Math.random(),
-        style: { fill: "white", stroke: lineColor, strokeWidth: thick }
-      };
-    });
+    const links = rootDescendents
```

Comment:
Any chance we can write a couple lines of comments about what this does? I can't imagine this is intuitive for first-time readers (it certainly isn't for me). Thanks! :-)

Reply:
done, added some comments about how the links are generated